src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -189,7 +189,10 @@ internal async Task ApplySearchParameterStatus(IReadOnlyCollection<ResourceSearc
                 updated.Add(param);
             }
 
-            _latestSearchParams = updatedSearchParameterStatus.Select(p => p.LastUpdated).Max();
+            if (updatedSearchParameterStatus.Any())
```

Comment:
>updatedSearchParameterStatus [](http://example.com/codeflow?start=16&length=28)

can we just assert it's not empty in first line of method and if it is, just leave?

Or it's necessary to run sync and process notification?

Reply:
You are correct, we can simply end if the collection is empty.